armi/materials/material.py

```diff
@@ -190,19 +190,13 @@ def linearExpansionPercent(self, Tk: float = None, Tc: float = None) -> float:
 
     def linearExpansionFactor(self, Tc: float, T0: float) -> float:
         """
-        Return a dL/L factor relative to T0 instead of to the material-dependent reference
-        temperature. This factor dL/Lc is a ratio and will be used in dimensions through the
-        formula::
+        Return a dL/L factor relative to T0 instead of the material-dependent reference
+        temperature.
 
-            dim = dim0*(1+dLL).
-
-        If there is no dLL, it should return 0.0
```

Comment:
Unless the statement on this line is no longer true, I think this line should stay.

Reply:
So this statement is a little strange to me. 

It's not necessarily _wrong_, but I think is trivial. I interpret it as "if there is no thermal expansion, the linear expansion factor will be 0.0". Which is obvious, no?

Reply:
I think it is important so that people don't create a new material and have it return `None` or something like that. 

We should be able to rely upon an actual float being returned, otherwise downstream things will break.

To me, that's the importance.

Reply:
https://github.com/terrapower/armi/pull/1204/commits/45ea38345dd50fa9e85d720a851c27704432b179